distuv/distribution_test.go

```diff
@@ -160,7 +168,51 @@ func checkPDFContinuous(t *testing.T, i int, x []float64, p probLogprober) {
 	}
 }
 
-func checkPDFDiscrete(t *testing.T, i int, xs []float64, p probLogprober, tol float64) {
+// checkProbCdfContinuous checks that the Prob, Rand, and Quantile are all consistent.
```

Comment:
Wouldn't it be clearer if the samples were generated anew inside each check*() function?